README.md

```diff
@@ -204,66 +206,112 @@ const toast = document.createElement('std-toast');
 console.log(toast.open); // false
 ```
 
+There will additionally be an `action` property,
+which returns or allows setting an element that provides the toast's action.
+In detail:
+
+- The getter returns the first `button` or `a` element descendant with the `slot="action"` attribute set.
+  If no such element exists,
+  it returns null.
+- The setter accepts a `button` or `a` element,
+  appends it to the toast element,
+  and sets the `slot="action"` attribute on it.
+  If any existing such elements exist,
+  the first one (i.e. the one that would be returned by the getter) is replaced.
+  If the provided value is not a `HTMLButtonElement` or `HTMLAnchorElement`,
+  a `TypeError` is thrown.
```

Comment:
Hmm, this is probably too restrictive given custom elements. Maybe better not to enforce anything in the setter. I will push another commit.